amun/inspect.py

```diff
@@ -88,6 +88,22 @@ def _gather_os_release():
     return result
 
 
+def _gather_runtime_environment(os_release: dict):
+    """Gather information about runtime environment."""
+    return {
+        "cuda_version": None,
+        "hardware": {
+            "cpu_family": None,
+            "cpu_model": None
+            },
+        "name": None,
```

Comment:
Name is not required - its used in thamos just to reference concrete entry.

Reply:
but there is actually missing `python_version`